app/frontend/templates/feed.html

```diff
@@ -0,0 +1,32 @@
+{% from "security/_macros.html" import form_field %}
+
+{% extends "base.html" %}
+
+{% block layout %}
+<div class="page header">
```

Comment:
need to put a '-' between page and header; otherwise you've set two different classes.